windows-server-container-tools/Install-ContainerHost/Install-ContainerHost.ps1

```diff
@@ -1168,12 +1168,12 @@ mkdir %ProgramData%\docker
 
 :run
 if exist %certs%\server-cert.pem (if exist %ProgramData%\docker\tag.txt (goto :secure))
- 
+
 docker daemon -D -b "$global:SwitchName"
 goto :eof
- 
+
 :secure
-docker daemon -D -b "$global:SwitchName" -H 0.0.0.0:2376 --tlsverify --tlscacert=%certs%\ca.pem --tlscert=%certs%\server-cert.pem --tlskey=%certs%\server-key.pem
+docker daemon -D -b "$global:SwitchName" -H npipe:////./pipe/docker_engine -H 0.0.0.0:2376 --tlsverify --tlscacert=%certs%\ca.pem --tlscert=%certs%\server-cert.pem --tlskey=%certs%\server-key.pem
```

Comment:
I've tested this with @taylorb-microsoft's TLS script today morning and it worked fine. TLS over 2376 from outside and probably no TLS with npipe to work within the TP4 machine locally. Tested this with a Mac Docker Client against the TP4 VM.

Works just like the Linux version:

```
/usr/local/bin/docker daemon -D -g /var/lib/docker -H unix:// -H tcp://0.0.0.0:2376 --label provider=vmwarefusion --tlsverify --tlscacert=/var/lib/boot2docker/ca.pem --tlscert=/var/lib/boot2docker/server.pem --tlskey=/var/lib/boot2docker/server-key.pem -s aufs
```